themes-default/slim/test/specs/name-pattern.spec.js

```diff
@@ -27,14 +27,13 @@ test('renders', t => {
         },
         methods: {
             checkNaming(pattern, selectedMultiEpStyle, animeType) {
-                return new Promise((resolve, reject) => {
+                return new Promise((resolve, reject) => { // eslint-disable-line no-unused-vars
                     console.debug(`Mocking method checkNaming with params: ${pattern}, ${selectedMultiEpStyle} and ${animeType}`);
                     resolve();
                 });
-                
             },
             testNaming(pattern, selectedMultiEpStyle, animeType) {
-                return new Promise((resolve, reject) => {
+                return new Promise((resolve, reject) => { // eslint-disable-line no-unused-vars
```

Comment:
Why not change this to `return new Promise(resolve => {` since you're not using `reject`?

Reply:
Your allowed to do that?

Reply:
I don't see why not, you can omit the params you're not using.